packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem/AccountItemLogo/AccountItemLogo.tsx

```diff
@@ -2,26 +2,29 @@ import { Column, Icon } from '@trezor/components';
 import { CoinLogo } from '@trezor/product-components';
 import { exhaustive } from '@trezor/type-utils';
 
-import { Account, AccountItemType } from '../../../../../types/wallet';
-import { TokenIconSetWrapper } from '../../../TokenIconSetWrapper';
+import { type Account, type AccountItemType } from '../../../../../../types/wallet';
```

Comment:
Please use absolute imports
```suggestion
import { type Account, type AccountItemType } from 'src/types/wallet';
```